src/main/java/org/prebid/server/auction/AmpRequestFactory.java

```diff
@@ -24,12 +30,24 @@
 
     private static final String TAG_ID_REQUEST_PARAM = "tag_id";
     private static final String DEBUG_REQUEST_PARAM = "debug";
+    private static final String OW_REQUEST_PARAM = "ow";
+    private static final String OH_REQUEST_PARAM = "oh";
+    private static final String W_REQUEST_PARAM = "w";
+    private static final String H_REQUEST_PARAM = "h";
+    private static final String MS_REQUEST_PARAM = "ms";
+    private static final String CURL_REQUEST_PARAM = "curl";
+    private static final String SLOT_REQUEST_PARAM = "slot";
+    private static final String TIMEOUT_REQUEST_PARAM = "timeout";
 
+    private final int timeoutAdjustmentMs;
     private final StoredRequestProcessor storedRequestProcessor;
     private final AuctionRequestFactory auctionRequestFactory;
 
-    public AmpRequestFactory(StoredRequestProcessor storedRequestProcessor,
-                             AuctionRequestFactory auctionRequestFactory) {
+    public AmpRequestFactory(
+            int timeoutAdjustmentMs,
```

Comment:
Intentionally made it an integer type, think it has sufficient range for adjustment value, but It's inconsistent with the rest of codebase, so if reviewer decides to change it I'll do it immediately. An additional reason for that is I'm treating incoming timeout param also as an integer to avoid a creation of parseLong... methods.